aspnet/mvc/views/forms/sample/final/Controllers/DemoRegisterController.cs

```diff
@@ -0,0 +1,71 @@
+﻿using FormsTH.ViewModels;
+using Microsoft.AspNet.Mvc;
+
+namespace FormsTH.Controllers
+{
+    // This controller is used only to demonstrate working with forms.
+    public class DemoRegController : Controller
```

Comment:
@dougbu I started out with DemonstrationRegistrationController per @blowdart suggestion (my other names were rejected for security reasons) - but the name was so long it made some of the code snippets have horizontal scroll bars. @blowdart  is it OK if I rename it RegistrationController ? We don't want customers to use this as starting code for a real registration controller.